docs/ReleaseNotes/Menes.v3.md

```diff
@@ -1,5 +1,12 @@
 # Release notes for Menes v3.
 
+## v3.2
+
+### v3.2.0
+
+Update `Microsoft.OpenApi.Readers from 1.2.3 to 1.3.2
```

Comment:
If you're including this, don't you need to also mention the other libraries you've updated?

Reply:
The others are patch updates, so they wouldn't have necessitated a new minor version of this package. We don't normally update this history for that sort of thing. (Most changes of that type are picked up by Dependabot, which doesn't edit our release notes for us.) So I generally only mention dependency updates that imply API changes (so either minor or major version changes).